espnet2/bin/s2t_inference.py

```diff
@@ -188,6 +188,7 @@ def __init__(
         lang_sym: str = "<eng>",
         task_sym: str = "<asr>",
         predict_time: bool = False,
+        skip_last_chunk_threshold: float = 0.2,
```

Comment:
I was thinking adding this argument in `decode_long` only, since it is specific to long-form decoding but never used in short-form `__call__`. If we remove it from `__init__`, we can also remove the argument in `inference` and `argparser`.

Reply:
Revised.